.github/workflows/npm-release.yml

```
@@ -0,0 +1,95 @@
+name: npm-release
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        type: choice
+        description: "Add node release."
+        required: true
+        options:
+          - patch
+          - minor
+          - major
+      app-path:
+        type: choice
+        description: Service-Name
+        required: true
+        options:
+          - refarch-frontend # todo: muss angepasst werden. eigenen Ordner verwenden
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # to be able to publish a GitHub release
+      issues: write # to be able to comment on released issues
+      pull-requests: write # to be able to comment on released pull requests
+      id-token: write # to enable use of OIDC for npm provenance
+    outputs:
+      ARTIFACT_NAME: ${{ steps.node.outputs.artifact-name }}
+      VERSION: ${{steps.node-release.outputs.VERSION}}
+    env:
+      skipDeployment: false
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "22"
+          registry-url: 'https://registry.npmjs.org'
+
+        
+      - id: node-release
+        name: Bump version and create git tag
+        working-directory: ./${{inputs.app-path}}
+        run: | 
+         NEW_VERSION=$(npm version ${{inputs.releaseVersion}}) || exit 1
+         echo "VERSION=$NEW_VERSION" >> "$GITHUB_OUTPUT"
+         git config --global user.email "github-actions@github.com"
+         git config --global user.name "GitHub Actions"
+         git add package.json package-lock.json
+         git commit -m "Bump version to ${NEW_VERSION}" || exit 1
+         git tag "${NEW_VERSION}" || exit 1
+         git push && git push --tags || exit 1
+      - id: node
+        uses: it-at-m/.github/.github/actions/action-npm-build@main
+        with:
+          app-path:  "${{ inputs.app-path }}"
+      - if: "${{ !env.skipDeployment }}"
+        run: npm --prefix ./${{ inputs.app-path }} publish   # publish
```

Remove irrelevant comment or better describe it
```suggestion
        run: npm --prefix ./${{ inputs.app-path }} publish
```